backend/src/main/resources/application-create-sample-data.yaml

```diff
@@ -1,13 +1,43 @@
 # configuration for dev and demo server startup on a blank DB
 simple-report-initialization:
-  organization:
-    org-name: Dis Organization
-    external-id: DIS_ORG
-  facility:
-    facility-name: Injection Site
-    clia-number: "000111222-3"
-    address:
-      street-1: 2797 N Cerrada de Beto
+  organizations:
+    - org-name: Dis Organization
+      external-id: DIS_ORG
+    - org-name: Dat Organization
+      external-id: DAT_ORG
+  facilities:
+    # TODO: uncomment once e2e tests can accommodate 2+ facilities
```

Comment:
can you suggest an alternative? not sure what you mean

Reply:
We currently have two lists that are tied together by repeating information (the org external ID) that has to appear in the right place in each item of each list. That mirrors the database, but is a bit of a maintenance and updating hassle—I think it would make more sense to have the data structure in YAML nest the facilities under the organization. That said, I think this was the right way to handle breaking the "only one org allowed" logjam, so that's a suggestion for later refactoring, not an objection to what you've done here.

Reply:
yeah, this actually dovetails well with a discussion i had with @nickscialli-usds earlier today: that his first instinct was to find the user's filtered facility list in `user.organization.testingFacility`, where they are not. they are in `user.facilities`, precisely because `user.organization` is bound to the `Organization` class that does not currently have `Facility`s as member objects. so the graphql layer currently just populates `user.organization.testingFacility` by using  just `organizationExternalId` to find the facility list (which felt like too much to refactor in this PR). but anyway, if there were some refactor of `Organization` to include facilities, we could make `user.organization.testingFacility` have the user's filtered list *and* refactor this config structure. but agreed that it's all better for another PR.

Reply:
That sounds like a separate issue that I should look at because I think I agree with Nick's first instinct, but that is not for this thread. 😁